woocommerce/woocommerce-android
# Glance to swipeable item

Closes: #7037

### Description
This PR adds a short glance animation to reveal the underlying swipe actions of a given order. As described in the [issue](#7037 ), the glance animation should only be displayed the first time we open the orders section. 
This glance animation was created with the help of `MotionEvent`, simulating a small swipe in the view. 

### Testing instructions

1. Open orders
2. Check that the first non-completed order animates

### Images/gif


https://user-images.githubusercontent.com/18119390/182453654-ca0ff63f-6fe8-4977-9184-ba1d014dbe42.mp4
